### PR TITLE
Add copy functionality

### PR DESCRIPTION
**Pull request by Giri-Spheron**
Updated to nextra's new version with copy option in code snippet

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
-const withNextra = require("nextra")({
-  theme: "nextra-theme-docs",
-  themeConfig: "./theme.config.tsx",
-});
+const withNextra = require("nextra")(
```
Comment (aayushmahapatra) [inline next.config.js:1]:
```suggestion
const withNextra = require("nextra")({
```

File: next.config.js [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+  "nextra-theme-docs",
+  "./theme.config.tsx",
+  {
```
Comment (aayushmahapatra) [inline next.config.js:4]: Remove this.

**Review by aayushmahapatra — CHANGES_REQUESTED** [review]
Some changes required. Also, make sure to format the code before pushing.

File: next.config.js (hunk 2 above)
Comment (Giri-Spheron) [reply next.config.js]: done changes

**Review by aayushmahapatra — APPROVED** [review]

**Pull request merged**